roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/configure-worker-node.yml

```diff
@@ -12,8 +18,20 @@
     index_var: __sap_hypervisor_node_preconfigure_register_worker_network_nr
   when: __sap_hypervisor_node_preconfigure_register_worker.networks is defined
 
+
 # How to wait for node to be scheduleable? (NodeSchedulable)
 - name: Wait for all k8s nodes to be ready
-  ansible.builtin.command: oc wait --for=condition=Ready nodes --all --timeout=3600s
-  register: __sap_hypervisor_node_preconfigure_register_nodes_ready
-  changed_when: __sap_hypervisor_node_preconfigure_register_nodes_ready.rc != 0
+  kubernetes.core.k8s_info:
```

Comment:
yes that will also work. But in case of multi node failure the way ansible loops works , i think it will keep on looping on one node till we hits timeout or it is ready and then goto next node

Reply:
There is a bug in wait_condition https://github.com/ansible-collections/kubernetes.core/issues/697